### PR TITLE
Add scripts to list and check images from the registry

### LINKED ISSUE #688: Include elemental-operator related images into rancher security radar
We should provide a list of elemental images. I'd assume this is the same list of images that is used for airgap. We could do something similar to https://github.com/rancher/rancher/releases/download/v2.8.2/rancher-images.txt

### PR DESCRIPTION
**Pull request by davidcassany**
Part of #688

### CODE REVIEW

**Comment by davidcassany** [comment]
I added the scripts in `.obs/helper_scripts` folder and those are quite related to OBS procedures and releases, so I though including them under `.obs` makes it clear that those are not meant to be used as development or deployment helpers.

File: .obs/helper_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## List Elemental images
+
+The script `listimages.sh` is meant to list the Elemental images referrenced
```
Comment (fgiudici) [inline .obs/helper_scripts/README.md:3]:
```suggestion
The script `listimages.sh` is meant to list the Elemental images referenced
```

File: .obs/helper_scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+in charts to stdout. As arguments one can provide the elemental-operator version and/or
+the registry URL to use a suffix for image refrences. For instance:
```
Comment (fgiudici) [inline .obs/helper_scripts/README.md:5]:
```suggestion
the registry URL to use a suffix for image references. For instance:
```

File: .obs/helper_scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+```
+$ ./listimages.sh -o 1.4.3 -r registry.opensuse.org/isv/rancher/elemental/dev/containers
+```
+
+By default uses the parent tag, if any, as the elemental-operator version and `registry.suse.com`
+as the default registry URL.
+
+The script is meant to be used generate text file listing all relavant images for a given
```
Comment (fgiudici) [inline .obs/helper_scripts/README.md:14]:
```suggestion
The script is meant to be used to generate text file listing all relevant images for a given
```

File: .obs/helper_scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+elemental-operator release.
+
+## Check Elemental images exist in registry
+
+The script `checkimages.sh` can be used to check the existance of a provided list of images.
```
Comment (fgiudici) [inline .obs/helper_scripts/README.md:19]:
```suggestion
The script `checkimages.sh` can be used to check the existence of a provided list of images.
```

**Review by fgiudici — APPROVED** [review]
LGTM! thanks for dropping a README too, that's really a good practice (feeling guilty... I should have done something similar with the airgap scripts 😓  )

**Pull request merged**